tests/ssg_test_suite/oscap.py

```diff
@@ -368,6 +368,13 @@ def run_stage(self, stage):
 
         if result:
             LogHelper.log_preloaded('pass')
+            if self.clean_files and stage in ['initial', 'final']:
```

Comment:
Seems that xml results are generated only in `initial` and `final` stage, but there are also verbose log files and these are generated in every stage and I think we should remove them too (`self.verbose_path`).

What about doing cleanup in every stage and have a condition if it is remediation stage only remove `self.verbose_path`?

Reply:
Ok, will try that.